server/server_test.go

```diff
@@ -181,6 +178,7 @@ func TestRunServer(t *testing.T) {
 }
 
 func TestListen(t *testing.T) {
+	// TODO(dmitris): check if the unused const 'name' can be deleted [PR3621]
```

Comment:
Can I get an update on this TODO? I'd prefer to not merge in TODOs if possible.

Reply:
I removed the unused variable and removed the `TODO` line in 5430f6c4.

Reply:
Strange, I still see the TODO in the latest version of the code, could you double check this?

Reply:
double-checked, no `TODO`s in `server/server_test.go` (or any files underneath `server/`) in the PR branch:
```bash
$ git log -1 && grep TODO server/server_test.go
commit ea94905c0a83e8c9db801f4b90a4e4a7524174da (HEAD -> golangci-lint, ds/golangci-lint)
Author: Dmitry S <dsavints@gmail.com>
Date:   Mon Apr 29 17:36:37 2024 +0200

    remove comment about error handling
$ rg TODO server/
$
```
[server/server_test.go](https://github.com/dmitris/prebid-server/blob/golangci-lint/server/server_test.go) in my / PR [branch](https://github.com/dmitris/prebid-server/blob/golangci-lint).